src/main/java/me/youhavetrouble/justchat/ConfigReload.java

```diff
@@ -0,0 +1,39 @@
+package me.youhavetrouble.justchat;
+
+import org.bukkit.plugin.Plugin;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ConfigReload {
```

Comment:
I don't like the name. This class holds config values, what does it have to do with reloading them? reload should just be a method reassigning them.